environments/.env.production

```diff
@@ -40,3 +40,4 @@ DAI_ADDRESS=0x6B175474E89094C44Da98b954EedeAC495271d0F
 USDC_ADDRESS=0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
 POOLTOGETHER_GRAPH_ID=QmSmtdcYD9Zuk7W9zuk73rRCNema26vMdF7muyfGwLzqfT
 THE_GRAPH_API_URL=https://api.thegraph.com/subgraphs/id
+SHOW_LANG_TOGGLE=
```

Comment:
Shall we assume this to be `true`?

Reply:
Nope, it should be false by default, as SHOW_LANG_TOGGLE manages development tool to toggle languages quickly on each screen :)

Reply:
Thanks!